web/test/pages/[taskUrlSlug].test.tsx

```diff
@@ -214,6 +214,32 @@ describe("task page", () => {
     expect(subject.queryByText(TaskDefaults.requiredTagText)).not.toBeInTheDocument();
   });
 
```

Comment:
A test that verifies neither display if they are both undefined, as well as the HR would also be good, since they are all surrounded by an if block